File: api_auditeur/src/main/java/com/example/api_auditeur/dto/InscriptionDto.java
package com.example.api_auditeur.dto;

import com.example.api_auditeur.model.page_enum.EtatInscription;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDate;

@Getter
@Setter
@AllArgsConstructor
public class InscriptionDto {

    private Long id;

    private Long formationId;
    private Long utilisateurId;
    private Long paiementId;

    private String dateInscription;
    private EtatInscription etatInscription;

    private String numeroCni;
    private String dateNaissance;
    private String address;
    private String numeroTel;

    private String fichier;
}

File: api_auditeur/api_auditeur_ang/src/app/models/inscriptiondto.ts
import { EtatInscription } from './inscriptions';

export interface InscriptionDto {

  id: number;
  formationId: number;
  utilisateurId: number;
  paiementId: number | null;

  dateInscription: string;
  etatInscription: EtatInscription;

  numeroCni: string;
  dateNaissance: string;
  address: string;
  numeroTel: string;

  fichier: string;
}
